src/RestControllers/AuthRestController.php

```diff
@@ -247,7 +247,7 @@ public function isValidToken($tokenRaw)
         } else { // $_SESSION['api'] == "oemr" || $_SESSION['api'] == "fhir"
             // For core api or fhir api
             $_SESSION['authUser'] = $tokenResult['username'];
-            $_SESSION['authUserId'] = $tokenResult['user_id'];
+            $_SESSION['authUserID'] = $tokenResult['user_id'];
```

Comment:
doh, that was likely my doing...

Reply:
I saw awhile back. Was driving me nuts.:)